.github/workflows/ci.yml

```diff
@@ -0,0 +1,172 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  pull_request_target:
+    branches: [ master ]
+    types: [ labeled ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: |
+      github.event_name != 'pull_request_target' &&
+      !contains(toJSON(github.event.commits.*.message), '[skip ci]') &&
+      !contains(toJSON(github.event.commits.*.message), '[ci skip]')
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12'
+
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: |
+          .yarn/cache
+          .pnp.js
+        key: yarn-deps-v1-${{ hashFiles('yarn.lock') }}
+        restore-keys: |
+          yarn-deps-v1-
+
+    - name: Install dependencies
+      run: yarn --immutable
+
+    - name: Lint code
+      run: yarn test:lint
+
+    - name: Run Flow
+      run: yarn test:flow
+      if: always()
+
+    - name: Run Jest tests
+      run: yarn test:jest --coverage
+      if: always()
+
+    - uses: codecov/codecov-action@v1
+      with:
+        directory: ./coverage/
+
+  screenshotter_dispatcher:
+    runs-on: ubuntu-latest
+    if: |
+      (github.event_name != 'pull_request_target' ||
+        (github.event.pull_request.head.repo.full_name != 'KaTeX/KaTeX' &&
+          contains(github.event.pull_request.labels.*.name, 'test screenshots'))) &&
+      !contains(toJSON(github.event.commits.*.message), '[skip ci]') &&
+      !contains(toJSON(github.event.commits.*.message), '[ci skip]')
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.result }}
+    
+    steps:
+    - id: set-matrix
+      uses: actions/github-script@v2
```

Comment:
This docs for this action say that it's possible to require external scripts.  If we did that we could lint the script.

Reply:
@kevinbarabash The repo has to be checked out in order to run the script on the repo. However, this job doesn't check out the repo, and adding `checkout` will increase the time by about 5 seconds, where this job takes only 1-2 seconds.

Reply:
I didn't realize that.  That's good to know.

Reply:
@kevinbarabash I'm creating a ESLint plugin, which lints JavaScript in GitHub Actions workflow. I'll enable it later.